src/Illuminate/Foundation/Vite.php

```diff
@@ -121,8 +143,8 @@ public function __invoke($entrypoints, $buildDirectory = 'build')
         $entrypoints = collect($entrypoints);
         $buildDirectory = Str::start($buildDirectory, '/');
 
-        if (is_file(public_path('/hot'))) {
-            $url = rtrim(file_get_contents(public_path('/hot')));
+        if (is_file(public_path($this->hotFilePath))) {
+            $url = rtrim(file_get_contents(public_path($this->hotFilePath)));
```

Comment:
I'm wondering if wrapping this in the `public_path` helper is to opinionated for the escape hatch.

Wondering if we should have a hot file path function that uses their value verbatim or our own and we set the property to `null` by default.

```php
protected function hotFilePath()
{
    return $this->hotFilePath ?? public_path($this->hotFilePath);
}
```

Reply:
I think it's a good idea. But I think then we should have one for the build directory as well? And if yes, then we need to touch the `__invoke` a bit more I guess.

Reply:
I think this should be the following, if the `$hotFilePath` is null by default.

```php
protected function hotFilePath()
{
    return $this->hotFilePath ?? public_path('/hot');
}
```